src/content/learn/manipulating-the-dom-with-refs.md

```diff
@@ -327,17 +330,39 @@ Pada contoh ini, `itemsRef` tidak menyimpan simpul DOM. Namun,menyimpan sebuah [
     const map = getMap();
     if (node) {
       // Add to the Map
-      map.set(cat.id, node);
+      map.set(cat, node);
     } else {
       // Remove from the Map
-      map.delete(cat.id);
+      map.delete(cat);
     }
   }}
 >
 ```
 
 Hal ini memungkinkan Anda membaca simpul DOM individu dari *Map* nanti.
 
+<Canary>
+
+This example shows another approach for managing the Map with a `ref` callback cleanup function.
```

Comment:
Does this change translate to this “PR Sync”? @resir014

Reply:
Yes, should be translated. @r17x